### ISSUE TITLE
JMAP support

### ISSUE THREAD

**Issue by zamentur** [YunoHost/issues#2333]
JMAP is a replacement protocol for IMAP.
https://jmap.io/software.html

An integration in the core or as a YunoHost app could be done, for example by using a JMAP proxy on top of an IMAP server.

You can get some explanation on it in french:
https://invidious.fdn.fr/watch?v=RO-msWBOSwE&list=PLU8kgF3be3rQByaL3UBArjy0E0SO7EZvK&index=12

**Comment by Tagadda** [comment]
I haven't found many proxy implementations yet and none seem to be actively maintained today.
- The original project, in perl https://github.com/jmapio/jmap-perl
- A python port https://github.com/filiphanes/jmap-proxy-python
- A more recent project in Rust https://github.com/stalwartlabs/imap-to-jmap/

**Comment by zamentur** [comment]
Here a comment from quentin working on aerogramme (a distributed IMAP server on top of garage) 
https://forum.chatons.org/t/aerogramme-serveur-imap-email/5446/13

TL;DR: he says EAS could be a good idea too.

**Comment by hermann-san** [comment]
Stalwart does support JMAP, as mentioned in the YH-forum [here](https://forum.yunohost.org/t/replace-yunohost-email-stack-with-stalwart/36768) and it has some further info on why JMAP is beneficial to have.
Sterna Mail is the Android App that supports JMAP. Thunderbird-Android wants to check it out as mentioned in the  [2026 Roadmap](https://roadmaps.thunderbird.net/de/android/)
